packages/js/components/src/pagination/page-arrows-with-picker.tsx

```diff
@@ -0,0 +1,121 @@
+/**
```

Comment:
This is the only item that is brand new, in order to match the new design.
I decided to create a new component instead of refactoring the `page-arrows.tsx` component, as that would still be used in other places.